src/main/resources/com/cloudbees/plugins/credentials/CredentialsSelectHelper/WrappedCredentialsStore/dialog.jelly

```diff
@@ -25,12 +25,9 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:l="/lib/layout">
   <l:ajax>
-    <div class="hd">
-      <h2 style="margin-top: 1rem" tooltip="${it.description}">${it.description}: ${it.displayName}</h2>
-    </div>
-    <div class="bd">
-      <h3>${%Add Credentials}</h3>
-      <form action="${it.url}/addCredentials" method="POST" id="credentials-dialog-form">
+    <div>
+      <form action="${it.url}/addCredentials" method="POST" id="credentials-dialog-form" data-title="${it.description}: ${it.displayName}" data-add="${%Add}">
```

Comment:
I think it would be better to have the `Add Credentials` in the title as well
e.g. `Add Credentials (Provider: Jenkins)`

Reply:
Good idea, but I'd suggest to do that in a follow-up PR and keep this one just about the design.